metals/src/main/scala/scala/meta/metals/Semanticdbs.scala

```diff
@@ -74,30 +75,53 @@ object Semanticdbs extends LazyLogging {
         }.toList
       case _ => Nil
     }
-    semanticdb.Document(input, "", Nil, messages, Nil, Nil)
+    m.Document(input, "", Nil, messages, Nil, Nil)
   }
 
+  implicit private val occurrenceOrdering: Ordering[SymbolOccurrence] =
+    new Ordering[semanticdb3.SymbolOccurrence] {
+      override def compare(x: SymbolOccurrence, y: SymbolOccurrence): Int = {
+        if (x.range.isEmpty) 0
+        else if (y.range.isEmpty) 0
```

Comment:
```if (x.range.isEmpty || y.range.isEmpty)` ?

Reply:
I made it this way to make it easier to see diff between `x.range` and `y.range`

Reply:
For that reason I like to format such conditions this way:

```scala
if (
  x.range.isEmpty || 
  y.range.isEmpty
) ...
```

But scalafmt with changes it to this: 

```scala
if (x.range.isEmpty || 
  y.range.isEmpty) ...
```

😒

Reply:
There's an undocumented `align.ifWhileOpenParen = true` to override that @laughedelic I agree that's the most annoying part of `align = false`, this is the indentation style used by Scala.js.

We should probably support config style in if conditions though 🤔